setup.py

```diff
@@ -54,7 +54,7 @@ def _single_compile(obj):
         src, ext = build[obj]
         self._compile(obj, src, ext, cc_args, extra_postargs, pp_opts)
     num_jobs = multiprocessing.cpu_count()
-    multiprocessing.pool.ThreadPool(num_jobs).map(_single_compile, objects)
+    list(multiprocessing.pool.ThreadPool(num_jobs).imap(_single_compile, objects))
```

Comment:
I wanted to force all the builds to finish before exiting; otherwise I get spew from the background processes over my command prompt.